osu.Game.Rulesets.Osu/Mods/OsuModStrictTracking.cs

```diff
@@ -25,7 +25,7 @@ public class OsuModStrictTracking : Mod, IApplicableAfterBeatmapConversion, IApp
         public override string Name => @"Strict Tracking";
         public override string Acronym => @"ST";
         public override ModType Type => ModType.DifficultyIncrease;
-        public override string Description => @"Once you start a slider, follow precisely or get a miss.";
+        public override string Description => @"Once you start a slider, follow precisely or get a sliderbreak.";
```

Comment:
```suggestion
        public override string Description => @"Once you start a slider, follow precisely or break combo.";
```

is probably what i would have gone with